### PR TITLE
added some styling for sandpack + theme

### PR DESCRIPTION
**Pull request by jordan-carlson99**
using nightOwl theme for now, if someone wants to change it just edit the import from @codesandbox/sandpack-react/themes and reference the ones found [here](https://sandpack.codesandbox.io/docs/getting-started/themes)

see comments in client editor for further styling

### CODE REVIEW

**Review by joegor85 — DISMISSED** [review]
Thanks for choosing the one I voted for, haha. Looks good to me.

File: client/src/components/LoginUI.jsx [hunk 1]
```diff
@@ -4,108 +4,156 @@ import { Row, Col, Container, Form, Button } from "react-bootstrap";
...
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    defaultValues: {
+      email: "",
+      password: "",
+    },
+  });
+
+  // const onSubmit = (data) => console.log(data);
```
Comment (CStewCodes) [inline client/src/components/LoginUI.jsx:22]: do we still need this?

File: client/src/components/LoginUI.jsx [hunk 2]
```diff
@@ -4,108 +4,156 @@ import { Row, Col, Container, Form, Button } from "react-bootstrap";
...
+              <Form.Label column sm={4}>
+                {" "}
+                Email address
+              </Form.Label>
+              <Col sm={8}>
+                <Form.Control
+                  type="email"
+                  placeholder="Enter email"
+                  // value={email}
+                  // I'm not sure why they did this? if there's a good reason to capture this every change i can re-do our submission
```
Comment (CStewCodes) [inline client/src/components/LoginUI.jsx:92]: I have zero idea about this.

**Review by DanielPurdy509 — APPROVED** [review]

**Review by CStewCodes — APPROVED** [review]
WELL DONE as always

**Pull request merged**